src/extension.ts

```diff
@@ -291,25 +291,34 @@ export function activate(context: vscode.ExtensionContext) {
 
         let editor = vscode.window.activeTextEditor;
         let glbPath = editor.document.uri.fsPath.replace('.gltf', '.glb');
-        let options = {
-            defaultUri: Uri.file(glbPath),
-            filters: {
-                'Binary glTF': ['glb'],
-                'All files': ['*']
-            }
-        };
-        vscode.window.showSaveDialog(options).then(uri => {
-            if (uri !== undefined) {
-                try {
-                    GlbExport.save(gltf, editor.document.uri.fsPath, uri.fsPath);
-                    vscode.window.showInformationMessage('Glb exported as: ' + uri.fsPath);
-                } catch (ex) {
-                    vscode.window.showErrorMessage(ex.toString());
+        if (!vscode.workspace.getConfiguration('glTF').get('neverPromptForFilename')) {
```

Comment:
Thanks for this!  Minor nitpick, would you mind renaming to something like `alwaysOverwriteDefaultFilename`?  Maybe that's a bit wordy but I'm looking for something that has a positive sense (where true means yes) and not a negative sense (where true means don't do it).  This request stems from some unrelated software I worked on that had an option called "dontSave" that would fail to save when set to true, and confused users for years.